### DISCUSSION TITLE
[Question][Hw4] About Grading

### DISCUSSION THREAD

**Discussion by jjhprogram** [prosyslab-classroom/cs348-information-security#573] [Q&A]
Hello. For grading Small test cases (50pt), will only example1.c and example2.c in the repository be used?
Or will there be extra examples for grading?

**Comment by Standchen** [comment]
In readme.md, it says,

> A set of small programs: "small" programs will be similar to the provided example programs in test that take string inputs.

So I guess there would be more small programs for grading.

**Comment by jjhprogram** [comment]
Thank you!
